index.js

```diff
@@ -1,6 +1,6 @@
 const util = require("util");
 const Discord = require("discord.js");
-const config = require("./config-dev.js");
+const config = require("./config.js");
```

Comment:
Create a symlink on your system so you do not have to change the code.

config.js is a symlink that will point to config-dev.js

Reply:
got it